schemas/RecipeFlow.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/RecipeFlow.json",
+    "title": "RecipeFlow",
+    "description": "The linkage between a recipe process, an action that structures a recipe, and a resource specification.",
```

Comment:
This description was hard to parse. And too geeky.
How about:
The specification of a resource input to, or output from, a recipe process.